frontend/src/components/TimerComponent/index.js

```diff
@@ -14,12 +15,20 @@ const CountdownTimer = ({ targetDate, sceneTime }) => {
   }, [targetDate]);
 
   const [days, hours, minutes, seconds] = useCountdown(time);
-
   if (days + hours + minutes + seconds <= 0) {
-    return <Notifcation setTime={setTime} sceneTime={sceneTime} />;
+    return <Notification setTime={setTime} sceneTime={sceneTime} />;
   }
+  return (
+    <div>
+      {days + hours + minutes + seconds > 25 &&
```

Comment:
you can try storing `days + hours + minutes + seconds` in a variable e.g. `time` or `totalTime` and use that in the conditional statements instead. It might make it easier to read 🙂